OMPython/__init__.py

```diff
@@ -447,3 +462,727 @@ def getClassNames(self, className=None, recursive=False, qualified=False, sort=F
+				return False
+		
+		else:
+			nonExistingList = []
+			for n in names:
+				if n in chkList:
+					pass
+				else:
+					isExist = False
+					nonExistingList.append(n)
+
+		if(isExist == False):
+			print 'Error!!! ', nonExistingList , ' does not exist.'
+			return False
+		else:
+			return True
+	
+	def getQuantities(self, names = None):
+		str_ = ''
+		strList = []
+		if (names == None):
+			nameList = []
+			startList =[]
+			changableList = []
+			variabilityList = []
+			descriptionList = []
+			
+			for q in self.quantitiesList:
+				nameList.append(q.name)
+				startList.append(q.start)
+				changableList.append(q.changable)
+				variabilityList.append(q.variability)
+				descriptionList.append(q.description)
+
+			dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}
+			frame = pd.DataFrame(dic)
+
+		else:
+			try:
+				checking = self.checkAvailability(names, self.qNamesList)
+				if(checking is False):
+					return None
+				
+				nameList = []
+				startList =[]
+				changableList = []
+				variabilityList = []
+				descriptionList = []
+				for n in names:
+					for q in self.quantitiesList:	
+						if n ==q.name:
+							nameList.append(q.name)
+							startList.append(q.start)
+							changableList.append(q.changable)
+							variabilityList.append(q.variability)
+							descriptionList.append(q.description)
+
+				dic = {'Name' : nameList, 'Value' : startList, 'Changable' : changableList, 'Variability' : variabilityList, 'Description' : descriptionList}	
+				frame = pd.DataFrame(dic)
+			except Exception as e:
+				return e
+		return frame
+
+	def getContinuousNames(self):
+		if self.cNamesList == []:
+			for l in self.quantitiesList:
+				if(l.variability == "continuous"):
+					self.cNamesList.append(l.name)
+		return self.cNamesList
+	
+	def getParameterNames(self):
+		if self.pNamesList == []:
+			for l in self.quantitiesList:
+				if(l.variability == "parameter"):
+					self.pNamesList.append(l.name)
+		return self.pNamesList
+		
+	def getInputNames(self):
+		if self.iNamesList == []:
+			for l in self.quantitiesList:
+				if(l.causality == "input"):
+					self.iNamesList.append(l.name)
+		return self.iNamesList
+		
+	def setInputSize(self):
+		size = len(self.inputs)
+		self.inputsVal = [None]*size		
+		
+	def getOutputNames(self):
+		if self.oNamesList == []:
+			for l in self.quantitiesList:
+				if(l.causality == "output"):
+					self.oNamesList.append(l.name)
+		return self.oNamesList
+		
+	def getContinuousValues_(self):
+		if self.cValuesList == []:
+			for l in self.quantitiesList:
+				if(l.variability == "continuous"):
+					self.cValuesList.append(l.start)
+		return self.cValuesList
+		
+	def getContinuousValues(self, contiName=None):
+		if contiName is None:
+			if self.cValuesList == []:
+				for l in self.quantitiesList:
+					if(l.variability == "continuous"):
+						str_ = l.start
+						if str_ is None:
```

Comment:
watch out when doing boolean comparison! `is` compares object identity, not content, so it's almost always more appropriate to use `==`, except when comparing to `None`.